README.md

```diff
@@ -141,3 +163,56 @@ bower install
 # Run Grunt to build the interface
 grunt
 ```
+
+## Deployment Steps
+
+1. Follow the steps to [edit settings in config.py as described
+   above](#edit-settings-in-configpy-file).
+1. Fork the [deployment repository](TODO: migrate repo to github) and then clone
+   that fork to a directory alongside your `adage-server` repository.
+
+   ```shell
+   > cd /<your chosen directory>/
+   > git clone git@github.com:<your github account>/adage-deploy.git
+   ```
+
+1. Install `fabric`, the tool we use for scripting deployment steps:
```

Comment:
Yes. Good idea!

- [ ] add link to the Fabric web site here (it can be had to find the right "fabric" via Google)